_topic_maps/_topic_map.yml

```diff
@@ -2198,6 +2198,8 @@ Topics:
     File: nodes-nodes-working
   - Name: Managing nodes
     File: nodes-nodes-managing
+  - Name: Managing Graceful Node Shutdown
```

Comment:
Small comment - wondering if this new "graceful node shutdown" content would be better in under the existing ["Working with nodes"](https://docs.openshift.com/container-platform/4.12/nodes/nodes/nodes-nodes-working.html) section? This existing section could be renamed tbh, maybe something like the below and then add your new content as the final section: 

```yaml
- Preparing nodes for shutdown and deleting them
  - ...  
  - Gracefully shutting down nodes 
```

Reply:
We agreed with dev that this whole section needs some reworking of the titles. Beyond the scope of PR.